### PR TITLE
Add Toronto office specific content

### PR DESCRIPTION
**Pull request by lmandel**

### CODE REVIEW

File: state.json [hunk 1]
```diff
@@ -71,6 +71,15 @@
...
           "https://status.mozilla.org/"
         ]
+      },
+      {
+        "name": "tor",
+        "commands": [
+          "http://yyz-conf-free.paas.allizom.org/",
+          "http://yyz-conf-free.paas.allizom.org/list",
+          "http://departures.moo.mx/index.html#mozilla-toronto",
+          "http://forecast.io/#/f/43.6474,-79.3942"
```
Comment (lonnen) [inline state.json]: We have a hacky way of making this present a little better on the TVs. I've modified the MV specific line to work for Toronto here:

`"content type=html content=\"<body style='height:100%;background:linear-gradient(#EAEFF2,#D4DDE4)'><iframe style='border:0;position:absolute;top:50%;left:50%;height:50%;width:30vw;height:30vh;transform:translate(-50%,-50%) scale(3);' src='http://forecast.io/embed/#lat=43.6474&lon=-79.3942&name=Mozilla Toronto&units=ca&color=#E66000'>\""`

Comment (lonnen) [inline state.json]: This gives near term forecast and weather alerts formatted to read well from 10 feet. You don't have to take it, but I think it's nicer than the default website.

screencap

**Comment by lonnen** [comment]
Let me know about the forecast.io thing, otherwise this is r+

File: state.json (hunk 1 above)
Comment (lmandel) [inline state.json]: I think this is a nice improvement. r+ Thanks!

Comment (lonnen) [inline state.json]: k. I'll apply it after I merge.

**Pull request merged**